templates/store/snap-details/_details.html

```diff
@@ -109,9 +109,13 @@ <h2 class="p-modal__title" id="modal-title"><i class="p-icon--warning" style="wi
     (Ownership verified) 
     <span class="p-tooltip--btm-right" aria-describedby="verified-explanation">
       <i class="p-icon--information"></i>
-      <span class="p-tooltip__message" role="tooltip" id="verified-explanation">The publisher has verified that they own this domain. 
-  It does not guarantee the Snap is an official upload from the 
-  upstream project.</span>
+      {% if developer_validation == VERIFIED_PUBLISHER %}
+        <span class="p-tooltip__message" role="tooltip" id="verified-explanation">The publisher has verified that they own this domain</span>
+      {% else %}
+        <span class="p-tooltip__message" role="tooltip" id="verified-explanation">The publisher has verified that they own this domain.
+It does not guarantee the Snap is an official upload
```

Comment:
With just this sentence being the difference, would it work to just wrap just this sentence in "if publisher is not verified" (instead of whole tooltip being difference inside if/else statement?

Reply:
Ah, true.

I guess you could try doing `{%- if … -%}…{%- endif -%}` as dashes prevent Jinja from rendering empty lines in place of its commands, but it may still we fragile in `pre`.